code/modules/mob/living/brain/robotic_brain.dm

```diff
@@ -33,7 +32,7 @@
 		to_chat(user, "<span class='notice'>You press your thumb on [src] and imprint your user information.</span>")
 		imprinted_master = user
 		return
-	if(brainmob && !brainmob.key && !searching && !is_posibrain)
+	if(brainmob && !brainmob.key && !searching && can_be_reinhabited)
```

Comment:
You accidently flipped from `!` to no `!` here

Reply:
Did I? I also changed it from false by default to true.